backend/apps/webui/main.py

```diff
@@ -64,6 +65,7 @@
 app.state.config = AppConfig()
 
 app.state.config.ENABLE_SIGNUP = ENABLE_SIGNUP
+app.state.config.ENABLE_USERNAME_PASSWORD_LOGIN = ENABLE_USERNAME_PASSWORD_LOGIN
```

Comment:
`ENABLE_LOGIN_FORM`

Reply:
Updated all references to `ENABLE_USERNAME_PASSWORD_LOGIN` to the suggested `ENABLE_LOGIN_FORM`.